README.md

```diff
@@ -8,6 +8,7 @@
 [![Jenkins LTS Release](https://img.shields.io/endpoint?url=https%3A%2F%2Fwww.jenkins.io%2Fchangelog-stable%2Fbadge.json)](https://www.jenkins.io/changelog-stable)
 [![Docker Pulls](https://img.shields.io/docker/pulls/jenkins/jenkins.svg)](https://hub.docker.com/r/jenkins/jenkins/)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3538/badge)](https://bestpractices.coreinfrastructure.org/projects/3538)
+[![Reproducible Builds](https://img.shields.io/badge/Reproducible_Builds-ok-green?labelColor=blue)](https://maven.apache.org/guides/mini/guide-reproducible-builds.html)
```

Comment:
```suggestion
[![Reproducible Builds](https://img.shields.io/badge/Reproducible_Builds-ok-green)](https://maven.apache.org/guides/mini/guide-reproducible-builds.html)
```

Suggestion: remove `labelColor=blue` so the badge is the same color as the other ones.

<img src="https://img.shields.io/badge/Reproducible_Builds-ok-green?labelColor=blue" />
<img src="https://img.shields.io/badge/Reproducible_Builds-ok-green" />